api-admin/src/main/java/bio/terra/pearl/api/admin/controller/dataimport/ImportController.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.pearl.api.admin.controller.dataimport;
+
+import bio.terra.pearl.api.admin.api.DataImportApi;
+import bio.terra.pearl.api.admin.service.AuthUtilService;
+import bio.terra.pearl.api.admin.service.enrollee.EnrolleeImportExtService;
+import bio.terra.pearl.core.model.admin.AdminUser;
+import jakarta.servlet.http.HttpServletRequest;
+import java.io.DataInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+
+@Controller
+public class ImportController implements DataImportApi {
+  private AuthUtilService authUtilService;
+  private HttpServletRequest request;
+  private EnrolleeImportExtService enrolleeImportExtService;
+
+  public ImportController(
```

Comment:
controller code is work-in-progress and will be updated when integrated with front-end